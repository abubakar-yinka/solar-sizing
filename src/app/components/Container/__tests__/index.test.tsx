import * as React from "react";
import * as renderer from "react-test-renderer";
import WithTheme from "../../../../styles/theme/ThemeProvider";
import Container from "..";

describe("Container component UI", () => {
  test("Should render properly", () => {
    const tree = renderer
      .create(
        <WithTheme>
          <Container>Solar Sizing</Container>
        </WithTheme>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
