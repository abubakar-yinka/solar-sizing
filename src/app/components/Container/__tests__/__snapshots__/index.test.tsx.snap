// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container component UI Should render properly 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "flex": 1,
        "justifyContent": "center",
      },
    ]
  }
>
  Solar Sizing
</View>
`;
